benchmark/buffers/buffer-read.js

```diff
@@ -33,11 +35,18 @@ function main(conf) {
   var fn = `read${conf.type}`;
 
   buff.writeDoubleLE(0, 0, noAssert);
-  var testFunction = new Function('buff', `
-    for (var i = 0; i !== ${len}; i++) {
-      buff.${fn}(0, ${JSON.stringify(noAssert)});
-    }
-  `);
+
+  var call;
+  if (fn === 'readIntLE' || fn === 'readIntBE') {
+    call = `buff.${fn}(0, 1, ${noAssert})`;
```

Comment:
It would probably be better if the `byteLength` passed to the function would be the size of the whole buffer.

Reply:
It is also likely a good idea to increase the general buffer size a tiny bit.